presto-main/src/test/java/io/prestosql/sql/analyzer/TestFeaturesConfig.java

```diff
@@ -140,6 +141,7 @@ public void testExplicitPropertyMappings()
                 .put("optimizer.push-table-write-through-union", "false")
                 .put("optimizer.dictionary-aggregation", "true")
                 .put("optimizer.push-aggregation-through-join", "false")
+                .put("optimizer.unwrap-casts", "false")
```

Comment:
Please put them always in the same order (ie always last or always eg after "optimize mixed distinct aggregations" or something). ideally, same order as in the FC.